src/Config.ts

```diff
@@ -4,6 +4,7 @@ export interface IConfig {
     serviceName: string;
     payhawkUrl: string;
     portalUrl: string;
+    substitutePortalUrl? : string;
```

Comment:
I don't really like the name of this variable. For me it'd better like this:

```suggestion
    portalRedirectUrl? : string;
```